oslc4j-client/src/main/java/org/eclipse/lyo/oslc4j/client/resources/OslcQueryResult.java

```diff
@@ -299,4 +299,46 @@ public void remove() {
 
         return result;
     }
+
+    /**
+     * Return the enumeration of queried results from this page
+     *
+     * @return member statements from current page.
+     */
+    public Iterable<Resource> getMembers() {
```

Comment:
What’s this method for?

Reply:
Maybe then using Guava will be better than writing your own iterator https://stackoverflow.com/a/3888901

Reply:
I didn't want to introduce anything new. I copied the existing method :-)